impeller/display_list/dl_golden_unittests.cc

```diff
@@ -48,5 +52,32 @@ TEST_P(DlGoldenTest, CanRenderImage) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(DlGoldenTest, RotateColorFilteredPath) {
```

Comment:
What I'd do is create another file called dl_aiks_golden_unittests.  It will match the same structure as DlGoldenTest but have the name AiksTests.  Then you can move files from aiks to there.

Reply:
It doesn't look like I can rename it like that. I'm going to have the golden harvester do a string replace.

Reply:
What problem did you run into?  It should work, I've already moved tests that way when I split up aiks_unittests.cc.

Reply:
> Its not the test names, its suite names. It needs to be AiksTest otherwise the goldens are invalidated. We can't have multiple AiksTests set up or we trip assertions.

We already have multiple `AiksTests`, checkout aiks_unittests.cc and aiks_blend_unittests.cc.  On monday let's sync up and look at it together.  You are probably just running into the the problem that this one is in the flutter namespace and the others are in the impeller namespace.

Reply:
Oh you know what.  That's probably not going to work because AiksTests doesn't have access to OpenPlaygroundHere(display_list).  We would have to copy that over, which isn't so bad.

If we want to go down the route you are considering in the latest patch I think the renaming of the suite should happen in the same file as the tests.  The `setUp` method could override the name.

Let's meet up later today to discuss the options.  We have to consider too what do we want to do with newly written tests, not just ones we've migrated from aiks.